### PR TITLE
separating integration from mm_mail_add_cat

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress

## Related Issues
relates: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5348)

### Continue of mm_mail_add_cat- part two:
Reintroduce the `Email` pack category. packs to include are Email gateways, email packs. e.g: Trustwave, CISCO ESA, Cisco SMA, Proofpoint TAP, Symantec Messaging, Gateway, MimeCast, FireEye EX, EWS O365, EWS v2, Gmail, POP3, Gmail Single User, MS Graph Mail, ArmorBlox, EmailCommunication, Exchange 2016 Complaince, IronScales, MailSender(New), PerceptionPoint.

## Minimum version of Cortex XSOAR
- [X] 6.0.0

## Does it break backward compatibility?
   - [X] No

### CODE REVIEW

File: Packs/Armorblox/ReleaseNotes/1_0_10.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
```
Comment (Shellyber) [inline Packs/Armorblox/ReleaseNotes/1_0_10.md:3]: Please remove all the unnecessary spaces in the release notes files

File: Packs/Armorblox/ReleaseNotes/1_0_9.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (Shellyber) [inline Packs/Armorblox/ReleaseNotes/1_0_9.md:1]: Please revert the changes in this file

File: Packs/Exchange2016_Compliance/Integrations/Exchange2016_Compliance/Exchange2016_Compliance.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ commonfields:
 deprecated: true
 name: Exchange 2016 Compliance Search
 display: Exchange 2016 Compliance Search (Deprecated)
-category: Messaging
+category: Email
```
Comment (Shellyber) [inline Packs/Exchange2016_Compliance/Integrations/Exchange2016_Compliance/Exchange2016_Compliance.yml:7]: This integration is deprecated.
No need to touch it

File: Packs/FireEyeEX/ReleaseNotes/2_0_12.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### FireEye Email Security
+- Updated the pack category to *Email*.
```
Comment (Shellyber) [inline Packs/FireEyeEX/ReleaseNotes/2_0_12.md:5]: Add the docker image rn also

File: Packs/Gmail/ReleaseNotes/1_3_3.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### Gmail
```
Comment (Shellyber) [inline Packs/Gmail/ReleaseNotes/1_3_3.md:4]: Add the docker update also

File: Packs/GmailSingleUser/ReleaseNotes/1_3_7.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Gmail Single User
+- Updated the pack category to *Email*.
```
Comment (Shellyber) [inline Packs/GmailSingleUser/ReleaseNotes/1_3_7.md:5]: Add the docker update also

File: Packs/MailListener_-_POP3/ReleaseNotes/2_0_2.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### MailListener - POP3
+- Updated the pack category to *Email*.
```
Comment (Shellyber) [inline Packs/MailListener_-_POP3/ReleaseNotes/2_0_2.md:3]: Add the docker update also

**Review by Shellyber — CHANGES_REQUESTED** [review]
Nice. some general notes
1. Update all the relevant release notes with the docker images update 
2. Remove the extra spaces in the release notes files

File: Packs/Armorblox/ReleaseNotes/1_0_10.md (hunk 1 above)
Comment (maimorag) [reply Packs/Armorblox/ReleaseNotes/1_0_10.md:3]: I looked at the rest of RL as well and removed all the unnecessary spaces + added docker updated when it was updated

**Review by Shellyber — APPROVED** [review]
Looks good to me 
Let's just verify with @ShirleyDenkberg and @bakatzir regarding the release notes.

File: Packs/Armorblox/ReleaseNotes/1_0_10.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
...
+##### Armorblox
+- Updated the pack category to *Email*.
```
Comment (ShirleyDenkberg) [inline Packs/Armorblox/ReleaseNotes/1_0_10.md:5]:
```suggestion
Updated the pack category to *Email*.
```

File: Packs/MailSenderNew/ReleaseNotes/1_0_9.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Mail Sender (New)
+- Updated the pack category to *Email*.
```
Comment (ShirleyDenkberg) [inline Packs/MailSenderNew/ReleaseNotes/1_0_9.md:5]:
```suggestion
Updated the pack category to *Email*.
```

**Comment by ShirleyDenkberg** [comment]
@Shellyber Review of Release Notes completed.
